Source/MLXNN/Quantized.swift

```diff
@@ -67,6 +68,21 @@ open class QuantizedLinear: Linear {
         self.freeze()
     }
 
+    /// Initializer meant for subclasses to provide arrays directly.
+    ///
+    /// ### See Also
+    /// - ``Linear/init(weight:bias:)``
+    public init(
+        weight: MLXArray, bias: MLXArray? = nil, scales: MLXArray, biases: MLXArray, groupSize: Int,
+        bits: Int
+    ) {
```

Comment:
Similar to the init on Linear:

```swift
public init(weight: MLXArray, bias: MLXArray? = nil) {
```

this allows subclasses to pass the pre-configured arrays -- this is used by `QLoRALinear`.